Fix/bootstrap turbo docs (#17564)
symfony/symfony-docs: frontend/encore/bootstrap.rst

        $('[data-toggle="popover"]').popover();
    });

> .. tip::

Reviewer: Maybe just a new short section to make it easy to find?

```
Using Bootstrap with Turbo
---------------------------
```

>     If you are using bootstrap with Turbo Drive, to allow your javascript to load on each page change, remember to wrap your javascript in a ``turbo:load`` event listener.

Reviewer: Add a line break here, after (approximately) the 75th character.

        $('[data-toggle="popover"]').popover();
    });

Using Bootstrap with Turbo
---------------------------

> If you are using bootstrap with Turbo Drive, to allow your javascript to load on each page change,

Reviewer: ```suggestion
If you are using bootstrap with Turbo Drive, to allow your JavaScript to load on each page change,
```

> remember to wrap your javascript in a ``turbo:load`` event listener.

Reviewer: ```suggestion
wrap the initialization in a ``turbo:load`` event listener:
```
